src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj

```diff
@@ -21,14 +21,24 @@
     <Reference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" />
     <Compile Include="$(SharedSourceRoot)CommandLineUtils\**\*.cs" />
   </ItemGroup>
+   
+  <!--  We include this here to ensure that the shared framework is built before we leverage it when running
+  the dev server. -->
+  <ItemGroup>
+    <ProjectReference
+      Include="$(RepoRoot)src\Framework\App.Runtime\src\Microsoft.AspNetCore.App.Runtime.csproj"
+      PrivateAssets="All"
+      ReferenceOutputAssembly="false"
+      SkipGetTargetFrameworkProperties="true" />
+  </ItemGroup>
```

Comment:
Assuming we copied this from somewhere? I only ask because I don't have any context if those additional attribute settings

Reply:
Yep -- I copied this from an earlier PR. It ensures that the `Runtime` project gets built before this one.